modules/recommended-requirements-central-services.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/acs-recommended-requirements.adoc
+:_content-type: CONCEPT
+[id="recommended-requirements-central-services_{context}"]
+= Central services (self-managed)
+
+Central services contain the following components:
+
+* Central
+* Scanner
+
+[NOTE]
+====
+If you are using ACS:CS, you do not need to look at requirements of central services, because they are managed by RedHat. You can look further down to look for requirements of Secured cluster services.
+====
+
+[NOTE]
+====
+For default resource requirements for the scanner, take a look at the xref:../installing/acs-default-requirements.adoc#default-requirements-central-services-scanner_acs-default-requirements[default resource requirements page].
```

Comment:
```suggestion
For default resource requirements for the scanner, see the default resource requirements page.
```

Reply:
See previous comment about xrefs in modules & the workaround we need to use if we need a link.